templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -28,6 +28,13 @@ contents:
     #storage_option = [
     #]
 
+    # file_locking is whether file-based locking will be used instead of
+    # in-memory locking
+    file_locking = false
+
+    # Path to the lock file.
+    file_locking_path = ""
```

Comment:
crio by default sets: `file_locking_path = "/run/crio.lock"` do we want that here too?

Reply:
Done!